railties/lib/rails/application/configuration.rb

```diff
@@ -210,6 +210,7 @@ def load_defaults(target_version)
             active_support.hash_digest_class = OpenSSL::Digest::SHA256
             active_support.key_generator_hash_digest_class = OpenSSL::Digest::SHA256
             active_support.remove_deprecated_time_with_zone_name = true
+            active_support.format_version = 7.0
```

Comment:
Should not it be `cache_format_version`? Also we need to document this new option in the configuring.md guide

Reply:
Yes, good catch, I'll correct it. I can add the documentation, but I'd rather get a 👍 first. No point writing doc if the PR won't get merged :p

Reply:
> we need to document this new option in the configuring.md guide

Done.